packages/fhir-converter/src/lib/handlebars-converter/handlebars-helpers.js

```diff
@@ -1154,7 +1165,7 @@ module.exports.external = [
       if (referenceData == undefined) {
         return "";
       }
-      return JSON.stringify(referenceData).slice(1, -1);
+      return JSON.stringify(referenceData).slice(1, -1).replace(/\s+/g, "").trim();
```

Comment:
The regex is to remove double spaces (it breaks schema)

Reply:
This removes all whitespaces from a string

`replace(/ {2,}/g, " ")` would replace doubles with singles